grpc/server/src/main/java/io/helidon/grpc/server/GrpcServerImpl.java

```diff
@@ -300,4 +334,84 @@ public void undeploy(BindableService service, String sName) {
     public ManagedChannel createInProcessChannel() {
         return InProcessChannelBuilder.forName(config.name()).build();
     }
+
+    /**
+     * Return an instance of SslContextBuilder from the specified SslConfig.
+     *
+     * @param  sslConfig the ssl configuration
+     *
+     * @Return an instance of SslContextBuilder
+     */
+    private SslContextBuilder getSslContextBuilder(SslConfiguration sslConfig) {
```

Comment:
This should probably be called something like `createSslContextBuilder` as it is not really a getter method, it is actually building a new instance of `SslContextBuilder`.

Reply:
Probably just `sslContextBuilder`, based on dev guidelines.